addons/format_lame.c

```diff
@@ -0,0 +1,522 @@
+/*
+ * Asterisk -- An open source telephony toolkit.
+ *
+ * MP3 write support using LAME
+ * added by Naveen Albert <asterisk@phreaknet.org>
+ * and sponsored by Marfox Ltd.
+ *
+ * MP3 read support using LAME
+ * added by Sperl Viktor <viktike32@gmail.com>
```

Comment:
Can you just use the traditional copyright header here? See `main/asterisk.c` for an example.

It would be great if one of the project owners could give their 2¢ on this so I can stop beating this drum if I'm in the wrong.

Reply:
New modules must use the traditional copyright header.